json-schemas/src/client-events-connections.json

```
@@ -76,6 +76,25 @@
       "type": "integer",
       "description": "The duration the connection was open for in milliseconds."
     },
+    "credentials": {
+      "type": "object",
+      "description": "The connection's token credentials.",
```

```suggestion
      "description": "Details of the credential used to authenticate the client.",
```

I say "Details of" since it isn't the actual credential, just details about it.